source/reporters/ut_tap_reporter.tpb

```diff
@@ -0,0 +1,124 @@
+create or replace type body ut_tap_reporter is
+
+
+  constructor function ut_tap_reporter(self in out nocopy ut_tap_reporter) return self as result is
+  begin
+    self.init($$plsql_unit);
+    self.lvl := 0;
+    return;
+  end ut_tap_reporter;
+
+  member procedure print_comment(self in out nocopy ut_tap_reporter, a_comment clob) as
+  begin
+    self.print_clob(regexp_replace(a_comment, '^', '# ', 1, 0, 'm'));
+  end print_comment;
+
+  member function escape_special_chars(self in out nocopy ut_tap_reporter, a_string_to_escape clob) return clob as
```

Comment:
Do you expect the strings here to be larger than 32k that we using CLOBs? I can see that you are using `testnames` etc there, so doesn't sound like needs `clob`.
If so for a sake of performance would suggest to change that to procedure with clob nocopy.
Running lot of temp clobs to for each line might not be best way,